### PR TITLE
bug fix 2962

### PR DESCRIPTION
**Pull request by hanykumar**
Hi, Issue: #2962, Buttons fixed.
Screenshot 2020-03-13 at 5 38 20 PM

### CODE REVIEW

File: src/theme/components/Button.js [hunk 1]
```diff
@@ -380,7 +380,8 @@ export default (variables /* : * */ = variable) => {
     shadowOpacity: platformStyle === PLATFORM.MATERIAL ? 0.2 : undefined,
     shadowRadius: platformStyle === PLATFORM.MATERIAL ? 1.2 : undefined,
     alignItems: 'center',
-    justifyContent: 'space-between'
+    justifyContent: 'space-between',
+    alignSelf: 'flex-start'
```
Comment (sankhadeeproy007) [inline src/theme/components/Button.js]: Can we do it without using alignSelf? The alignment of the button should be left to the user.

Comment (hanykumar) [reply src/theme/components/Button.js]: I think `alignSelf` is fine. I don't find any other solution. The alignment of the button is left to the user. Isn't it?

Comment (sankhadeeproy007) [reply src/theme/components/Button.js]: I mean the user might want to align it to the center or to the right. So I think it should be the user's choice, can we use `alignSelf: 'auto'` and see if that works?

Comment (hanykumar) [reply src/theme/components/Button.js]: `alignSelf: 'auto' `will set width to full. But User always has a choice for `alignSelf: 'center'` whether we set `alignSelf: 'flex-start'` by default or not.

**Pull request merged**